fetch/api/basic/scheme-about.any.js

```diff
@@ -16,3 +16,11 @@ checkNetworkError("about:blank", "POST");
 checkNetworkError("about:invalid.com");
 checkNetworkError("about:config");
 checkNetworkError("about:unicorn");
+
+promise_test(function(test) {
+  var promise = fetch("about:blank", {
+    "method": "GET",
+    "Range": "bytes=1-10"
+  });
+  return promise_rejects_js(test, TypeError, promise);
+}, "Fetching about:blank with range header is KO");
```

Comment:
It would be slightly clearer to state that the `Range` header doesn't affect it I think.